packages/@sanity/components/sanity.json

```diff
@@ -829,6 +829,14 @@
       "path": "portal/Sticky.js"
     },
     {
+      "name": "part:@sanity/components/utilities/scrollContainer",
+      "description": "Portal that sticks to scrollContainer"
+    },
+    {
+      "implements": "part:@sanity/components/utilities/scrollContainer",
```

Comment:
Can we stick to the `kebab-case` naming convention for part names?

Reply:
🇹🇷